Symfony/CS/Test/AccessibleObject.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Test;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
```

Comment:
Don't you want this class to be marked as `internal` as well? I don't think you want to support uses of this class outside of the project?